integration/integration_test.go

```diff
@@ -4858,7 +4858,7 @@ func testX11Forwarding(t *testing.T, suite *integrationTestSuite) {
 								}
 								return false
 							}, time.Second, 100*time.Millisecond, "failed to read display")
-						}, 10*time.Second, time.Second)
+						}, 10*time.Second, 2*time.Second)
```

Comment:
isn't it better to just use a select statement instead of increasing the time? Is 2s enough?

select{
case display <- strings.TrimSpace(string(output)):
default:
return true

}

?

Reply:
Yeah we can go with the select case, wasn't sure what's better